<launch>
    <include file="$(find rr_ml)/launch/run_seg.launch">
        <arg name="image_topic" value="/camera/image_color_rect"/>
        <arg name="detection_topic" value="/detection_image"/>
        <arg name="model" value="iarrc_lines"/>
    </include>
</launch>
